### PR TITLE
Version 0.1

### PR DESCRIPTION
**Pull request by mecha**
Initial release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Espresso Interface
 
+[![Build Status](https://travis-ci.org/Dhii/espresso-interface.svg?branch=master)](https://travis-ci.org/Dhii/espresso-interface)
+
```
Comment (XedinUnknown) [inline README.md:6]: What about CodeClimate?

**Review by XedinUnknown — CHANGES_REQUESTED** [review]
Should `ContextInterface` extend `ValueAwareInterface`?

File: README.md (hunk 1 above)
Comment (mecha) [reply README.md:6]: Didn't yet add the repo to CodeClimate. Currently checking if we can share administration of repos registered in CC.

**Comment by mecha** [comment]
Yes. In fact, it should already be extending it 😕

Which is odd because dependents that extend `EvaluableInterface` depend on `ValueAwareInterface` not `ContextInterface` ... and yet their tests pass.

File: README.md (hunk 1 above)
Comment (XedinUnknown) [reply README.md:6]: I added everything

**Comment by XedinUnknown** [comment]
@mecha, I know, right!? I saw code that depends on `ValueAwareInterface`. Dunno why this isn't there.

**Comment by XedinUnknown** [comment]
I'm on Gitter, by the way, @mecha, but you haven't replied since yesterday...

**Review by XedinUnknown — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -1,37 +1,28 @@
...
     "authors": [
         {
-            "name": "Dhii",
+            "name": "Dhii Team",
             "email": "development@dhii.co"
         }
     ],
     "minimum-stability": "dev",
     "require": {
         "php": "^5.3.9 | ^7.0",
```
Comment (XedinUnknown) [inline composer.json:14]: Is there a reason why min requirement is specifically PHP 5.3.*9*? I didn't find cases of method overriding. With the exception of [`ContextInterface#getValue()`](https://github.com/Dhii/espresso-interface/blob/master/src/ContextInterface.php#L21), which I don't know why it is there. Could you explain please?

Comment (mecha) [reply composer.json:14]: I usually use `5.3.9` in interface packages regardless of whether there is method overriding or not.  Probably a bad habit of mine.

`ContextInterface#getValue()` seems to be a remnant of when `ContextInterface` didn't extend `ValueAwareInterface`. Looks like I forgot about it.

Comment (XedinUnknown) [reply composer.json:14]: @mecha, one way to fix this could be to file a new PR for the release branch, and merge again after the fix.

**Comment by XedinUnknown** [comment]
@mecha, following our discussion on Gitter, please re-name project to `dhii/expression-interface`, and re-factor the code.

**Pull request merged**